personal_graph/__init__.py

```diff
@@ -1,3 +1,15 @@
+from .graph import Graph
 from personal_graph import graph
+from .ml import to_networkx, from_networkx
```

Comment:
this is an antipattern. Prefer absolute imports: `from personal_graph.ml import to_networkx, from_networkx`